src/sentry/identity/pipeline.py

```diff
@@ -53,6 +54,14 @@ def finish_pipeline(self):
             messages.SUCCESS,
             IDENTITY_LINKED.format(identity_provider=self.provider.name),
         )
+        metrics.incr(
```

Comment:
I think there are a few integrations that uses this pipeline class. Does it make sense to be including them in `login-success`?

Reply:
Oh i see...

So this is still used for social login
except it's embedding the social login into other custom integrations 🤔 

So the login-success would still be relevant, except we may want specific tags here.

one moment

Reply:
Ok.

I'll update the stats to reflect hte names of the classes they belong in.
Just makes it explicit where these stats are coming from and we can just decipher their use cases in the dashboards